front/src/applications/editor/tools/rangeEdition/speedSection/SpeedSectionMetadataForm.tsx

```diff
@@ -30,31 +30,49 @@ const getNewSpeedLimitTag = (
 const SpeedSectionMetadataForm: FC = () => {
   const { t } = useTranslation();
   const {
-    state: { entity },
+    state: { entity, error },
     setState,
   } = useContext(EditorContext) as ExtendedEditorContextType<RangeEditionState<SpeedSectionEntity>>;
 
-  const [uniqueIds, setUniqueIds] = useState<string[]>(
-    Object.keys(entity.properties?.speed_limit_by_tag || {}).map(() => nextId())
+  const [tagSpeedLimits, setTagSpeedLimits] = useState<
+    { id: string; tag: string; value: number | undefined }[]
```

Comment:
Or maybe we could extract this in a type ?

```suggestion
    { id: string; tag: string; value?: number }[]
```